_faq/supported-devices.html

```diff
@@ -0,0 +1,10 @@
+---
+categories:
+- player
+type: info
+title: "Supported devices"
+---
+<section>
+    <p>NewPipe supports Android phones and tablets, as well as TVs running Android/Google TV.</p>
+    <p>On some devices, it may be beneficial to change the appearance manually. This can be done by changing Settings &gt; Appearance &gt; List view model, which can be toggled between <tt>auto</tt>, <tt>list</tt> and <tt>grid</tt>.</p>
```

Comment:
```suggestion
    <p>On some devices, it may be beneficial to change the appearance manually. This can be done by changing <code class="code-bg-light"><img src="/img/icons/baseline-settings-20px.svg" class="baseline-settings" /> Settings > 
       <img src="/img/icons/color_lens-20px.svg" class="baseline-settings" /> Appearance > List view model</code>, which can be toggled between <code>Auto</code>, <code>List</code>, <code>Grid</code> and <code>Card</code>.</p>
```